src/qml/Mysthea/Pages/MainMenu.qml

```diff
@@ -13,34 +13,17 @@ Page {
     property bool hasToolbarLine: false
 
     signal cardsReferenceClicked
```

Comment:
here we will have 3 different signals: `mystheaClicked` , `icaionClicked`, `theFallClicked`.